numpy/typing/tests/data/reveal/strings.pyi

```diff
@@ -1,132 +1,201 @@
+
 assert_type(np.strings.rfind(AR_U, "a", start=[1, 2, 3]), npt.NDArray[np.int_])
-assert_type(np.strings.rfind(AR_S, [b"a", b"b", b"c"], end=9), npt.NDArray[np.int_])
+assert_type(np.strings.rfind(AR_B, [b"a", b"b", b"c"], end=9), npt.NDArray[np.int_])
+assert_type(np.strings.rfind(AR_S, AR_S, start=[1, 2, 3]), npt.NDArray[np.int_])
 
 assert_type(np.strings.index(AR_U, "a", start=[1, 2, 3]), npt.NDArray[np.int_])
-assert_type(np.strings.index(AR_S, [b"a", b"b", b"c"], end=9), npt.NDArray[np.int_])
+assert_type(np.strings.index(AR_B, [b"a", b"b", b"c"], end=9), npt.NDArray[np.int_])
+assert_type(np.strings.index(AR_S, AR_S, start=[1, 2, 3]), npt.NDArray[np.int_])
+
 assert_type(np.strings.rindex(AR_U, "a", start=[1, 2, 3]), npt.NDArray[np.int_])
-assert_type(np.strings.rindex(AR_S, [b"a", b"b", b"c"], end=9), npt.NDArray[np.int_])
+assert_type(np.strings.rindex(AR_B, [b"a", b"b", b"c"], end=9), npt.NDArray[np.int_])
+assert_type(np.strings.rindex(AR_S, AR_S, start=[1, 2, 3]), npt.NDArray[np.int_])
 
 assert_type(np.strings.isalpha(AR_U), npt.NDArray[np.bool])
+assert_type(np.strings.isalpha(AR_B), npt.NDArray[np.bool])
 assert_type(np.strings.isalpha(AR_S), npt.NDArray[np.bool])
 
 assert_type(np.strings.isalnum(AR_U), npt.NDArray[np.bool])
+assert_type(np.strings.isalnum(AR_B), npt.NDArray[np.bool])
 assert_type(np.strings.isalnum(AR_S), npt.NDArray[np.bool])
 
 assert_type(np.strings.isdecimal(AR_U), npt.NDArray[np.bool])
+assert_type(np.strings.isdecimal(AR_S), npt.NDArray[np.bool])
 
 assert_type(np.strings.isdigit(AR_U), npt.NDArray[np.bool])
+assert_type(np.strings.isdigit(AR_B), npt.NDArray[np.bool])
 assert_type(np.strings.isdigit(AR_S), npt.NDArray[np.bool])
 
 assert_type(np.strings.islower(AR_U), npt.NDArray[np.bool])
+assert_type(np.strings.islower(AR_B), npt.NDArray[np.bool])
 assert_type(np.strings.islower(AR_S), npt.NDArray[np.bool])
 
 assert_type(np.strings.isnumeric(AR_U), npt.NDArray[np.bool])
+assert_type(np.strings.isnumeric(AR_S), npt.NDArray[np.bool])
 
 assert_type(np.strings.isspace(AR_U), npt.NDArray[np.bool])
+assert_type(np.strings.isspace(AR_B), npt.NDArray[np.bool])
 assert_type(np.strings.isspace(AR_S), npt.NDArray[np.bool])
 
 assert_type(np.strings.istitle(AR_U), npt.NDArray[np.bool])
+assert_type(np.strings.istitle(AR_B), npt.NDArray[np.bool])
 assert_type(np.strings.istitle(AR_S), npt.NDArray[np.bool])
 
 assert_type(np.strings.isupper(AR_U), npt.NDArray[np.bool])
+assert_type(np.strings.isupper(AR_B), npt.NDArray[np.bool])
 assert_type(np.strings.isupper(AR_S), npt.NDArray[np.bool])
 
 assert_type(np.strings.str_len(AR_U), npt.NDArray[np.int_])
+assert_type(np.strings.str_len(AR_B), npt.NDArray[np.int_])
 assert_type(np.strings.str_len(AR_S), npt.NDArray[np.int_])
+
+assert_type(np.strings.translate(AR_U, AR_U), npt.NDArray[np.str_])
+assert_type(np.strings.translate(AR_B, AR_B), npt.NDArray[np.bytes_])
+assert_type(np.strings.translate(AR_S, AR_S), np.ndarray[np_t._Shape, np.dtypes.StringDType])
```

Comment:
I'm guessing that this red thingy is here because there's a trailing newline missing or something 🤔